e2e/playwright/viewer/src/tests/viewer-action.spec.ts

```diff
@@ -0,0 +1,164 @@
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { ApiClientFactory, getUserState, test, TEST_FILES, timeouts, Utils } from '@alfresco/playwright-shared';
+
+test.use({ storageState: getUserState('hruser') });
+test.describe('viewer action file', () => {
+  const apiClientFactory = new ApiClientFactory();
+  const randomFolderName = `playwright-folder-${Utils.random()}`;
+  const randomDocxName = `${TEST_FILES.DOCX.name}-${Utils.random()}`;
+  const randomDocxNameFavorite = `${TEST_FILES.DOCX.name}-${Utils.random()}`;
+  const randomDocxNameShare = `${TEST_FILES.DOCX.name}-${Utils.random()}`;
+  const randomDocxDelete = `${TEST_FILES.DOCX.name}-${Utils.random()}`;
+  const fileForEditOffline = `playwright-file1-${Utils.random()}.docx`;
+  const fileForCancelEditing = `playwright-file2-${Utils.random()}.docx`;
+  let folderId: string;
+  let fileDocxShareId: string;
+
+  test.beforeAll(async ({ fileAction, shareAction }) => {
+    await apiClientFactory.setUpAcaBackend('hruser');
+    const node = await apiClientFactory.nodes.createNode('-my-', { name: randomFolderName, nodeType: 'cm:folder', relativePath: '/' });
+    folderId = await node.entry.id;
+    await fileAction.uploadFile(TEST_FILES.DOCX.path, fileForCancelEditing, folderId);
+    await fileAction.uploadFile(TEST_FILES.DOCX.path, randomDocxName, folderId);
+    await fileAction.uploadFile(TEST_FILES.DOCX.path, randomDocxDelete, folderId);
+    const fileDocShare = await fileAction.uploadFile(TEST_FILES.DOCX.path, randomDocxNameShare, folderId);
+    fileDocxShareId = await fileDocShare.entry.id;
+    await fileAction.uploadFile(TEST_FILES.DOCX.path, randomDocxNameFavorite, folderId);
+    await shareAction.shareFileById(fileDocxShareId);
+    await fileAction.uploadFile(TEST_FILES.DOCX.path, fileForEditOffline, folderId);
+  });
+
+  test.beforeEach(async ({ personalFiles }) => {
+    const gotoNodeURL = `#/personal-files/${folderId}`;
```

Comment:
You don't need to create this variable, you can just pass the url directly into `navigate`